lib/utilities/aclUtils.js

```diff
@@ -117,7 +117,7 @@ aclUtils.parseAclXml = function parseAclXml(toBeParsed, log, next) {
 
         if (!result.AccessControlPolicy.Owner
```

Comment:
This is a bit out of scope but since we are touching the property, do you mind changing this to
```js
Array.isArray(result.AccessControlPolicy.Owner)
```
This way it's guaranteed that result.AccessControlPolicy.Owner is an array with at least one element in it per the conditions.